react/tsconfig.json

```diff
@@ -9,16 +9,14 @@
     "noImplicitAny": true,
     "noImplicitReturns": true,
     "noImplicitThis": true,
-    "noUnusedLocals": false,
-    "noUnusedParameters": false,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
```

Comment:
Nice!

Reply:
I actually don't like turning these on :-(

because during development I find them to be such a hassle, because I may leave intermediate variables unused just to test something but the build won't allow them